addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/InputWithPlayControl.java

```diff
@@ -13,24 +13,32 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import org.openhab.binding.yamahareceiver.YamahaReceiverBindingConstants;
+import static java.util.stream.Collectors.toSet;
+import static org.openhab.binding.yamahareceiver.YamahaReceiverBindingConstants.*;
 
 /**
  * The play controls protocol interface
  *
  * @author David Graeff - Initial contribution
- * @author Tomasz Maruszak - Spotify support
+ * @author Tomasz Maruszak - Spotify support, adding Server to supported preset inputs
  */
 
 public interface InputWithPlayControl extends IStateUpdatable {
     /**
      * List all inputs that are compatible with this kind of control
      */
     Set<String> SUPPORTED_INPUTS = Stream
-            .of(YamahaReceiverBindingConstants.INPUT_TUNER, "NET_RADIO", "NET RADIO", "USB", "DOCK", "iPOD_USB", "PC",
-                    "Napster", "Pandora", "SIRIUS", "Rhapsody", YamahaReceiverBindingConstants.INPUT_BLUETOOTH, "iPod",
-                    "HD_RADIO", YamahaReceiverBindingConstants.INPUT_SPOTIFY)
-            .collect(Collectors.toSet());
+            .of(
+                // Note (TM): Not sure why we have NET_RADIO, we should only have one 'NET RADIO' (as the canonical input name)
+                INPUT_NET_RADIO, "NET_RADIO",
```

Comment:
NET_RADIO was the original input, that worked on the RX-V671. "NET RADIO" will work as well in the current addon, I guess.